vaadin-form-layout-flow-parent/vaadin-form-layout-flow/src/main/java/com/vaadin/flow/component/formlayout/FormLayout.java

```diff
@@ -206,14 +220,56 @@ public FormItem(Component... components) {
             add(components);
         }
 
-        @Override
+        /**
+         * Adds the given components as children of this component at the slot
+         * 'label'.
+         *
+         * @param components
+         *            The components to add.
+         * @see <a href=
+         *      "https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot">MDN
+         *      page about slots</a>
+         * @see <a href=
+         *      "https://html.spec.whatwg.org/multipage/scripting.html#the-slot-element">Spec
+         *      website about slots</a>
+         */
+        protected void addToLabel(Component... components) {
```

Comment:
Does this method have any value?

Reply:
Yes, it's used here: 
https://github.com/vaadin/flow-components/blob/b64aac94dbbffed372c87683b1c1c973b91690ef/vaadin-form-layout-flow-parent/vaadin-form-layout-flow/src/main/java/com/vaadin/flow/component/formlayout/FormLayout.java#L383-L388